src/SnackSession.js

```diff
@@ -292,6 +300,94 @@ export default class SnackSession {
     return url;
   };
 
+  /**
+   * Generates a default app.json template for the snack.
+   * @returns {Object} the app.json template
+   * @function
+   */
+  generateAppJson = (): Object => {
+    const appJsonTemplate = {
+      expo: {
+        description: this.description,
+        slug: 'snack-' + this.snackId,
+        privacy: 'unlisted',
+        sdkVersion: this.sdkVersion,
+        version: '1.0.0',
+        orientation: 'portrait',
+        primaryColor: '#cccccc',
+        icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+        loading: {
+          icon: 'https://d1wp6m56sqw74a.cloudfront.net/~assets/c9aa1be8a6a6fe81e20c3ac4106a2ebc',
+          hideExponentText: false,
+        },
+        packagerOpts: {
+          assetExts: ['ttf', 'mp4', 'otf'],
+        },
+        ios: {
+          supportsTablet: true,
+        },
+      },
+    };
+    return appJsonTemplate;
+  };
+
+  /**
+   * Builds an apk from the snack and returns a url to download the apk.
+   * @param 
+   * @returns {Promise.<void>} A promise that contains the url when fulfilled.
+   * @function
+   */
+  getADKUrlAsync = async (
+    appJson : Object,
+    opts: {
+      current?: boolean,
```

Comment:
isCurrent ? shouldMakeCurrent ?
It's not obvious to me what current, mode, expIds, or type mean or what valid values would look like

Reply:
removed.